src/index.ts

```diff
@@ -65,6 +68,7 @@ let currentTraceListener: TraceListener | undefined;
  */
 export function datadog<TEvent, TResult>(
   handler: Handler<TEvent, TResult>,
+  forceWrap = false,
```

Comment:
Can we add forceWrap to the config object instead? Changing the signature of this handler means any customer using the config parameter will have to update their code. It also means we have to go in and update the serverless-plugin to use the new property.